src/main/java/org/broadinstitute/hellbender/utils/codecs/table/TableFeature.java

```diff
@@ -1,26 +1,27 @@
 package org.broadinstitute.hellbender.utils.codecs.table;
 
 
+import htsjdk.samtools.util.Locatable;
 import htsjdk.tribble.Feature;
-import org.broadinstitute.hellbender.utils.GenomeLoc;
-import org.broadinstitute.hellbender.utils.Utils;
 
 import java.util.List;
 
+import static org.broadinstitute.hellbender.utils.Utils.join;
```

Comment:
fine! i'll remove it here. 
But I'm static importing `Math` - the line is drawn there.